### PR TITLE
iZUMi TVL update: separate TVL on Ethereum, Polygon, and Arbitrum

### PR DESCRIPTION
**Pull request by McGrady00H**
We count the TVL on Polygon and Arbitrum besides the assets on Ethereum.  We separate the TVL on Ethereum, Polygon, and Arbitrum.  Thanks !

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/izumi/index.js exports TVL: 
                
 
 
arbitrum                  1.49 M
ethereum                  12.60 M
polygon                   3.83 M

total                    17.92 M

File: projects/izumi/index.js [hunk 1]
```diff
@@ -15,6 +33,15 @@ async function pool2() {
...
+    ethereum: {
+        fetch: fetchEthereum,
+    },
+    polygon: {
+        fetch: fetchPolygon,
+    },
+    arbitrum: {
+        fetch: fetchArbitrum,
+    },
+    pool2,
```
Comment (waynebruce0x) [inline projects/izumi/index.js:45]: can this be changed to 
pool2: {
 fetch: pool2
}
please

Comment (McGrady00H) [reply projects/izumi/index.js]: Done~

**Comment by llamatester** [comment]
The adapter at projects/izumi/index.js exports TVL: 
                
 
 
arbitrum                  1.49 M
pool2                     1.23 M
polygon                   3.84 M
ethereum                  12.67 M

total                    17.99 M

**Pull request merged**